vm/actor/src/builtin/puppet/mod.rs

```diff
@@ -204,14 +197,12 @@ impl ActorCode for Actor {
                 let res = Self::return_marshal_cbor_failure(rt)?;
                 Ok(Serialized::serialize(res)?)
             }
-
             Some(Method::RuntimeTransactionMarshalCBORFailure) => {
                 check_empty_params(params)?;
                 Self::runtime_transaction_marshal_cbor_failure(rt)?;
                 Ok(Serialized::default())
             }
-
-            _ => Err(rt.abort(ExitCode::SysErrInvalidMethod, "Invalid method")),
+            None => Err(rt.abort(ExitCode::SysErrInvalidMethod, "Invalid method")),
```

Comment:
can you also remove the rt.abort as I commented in favour of the macro? I want to remove this function after all actors are updated